client/blocks/reader-post-options-menu/index.jsx

```diff
@@ -251,8 +256,8 @@ class ReaderPostOptionsMenu extends React.Component {
 
 export default connect(
 	( state, ownProps ) => {
-		const feedId = ownProps.post.feed_ID;
-		const siteId = ownProps.post.is_external ? null : ownProps.post.site_ID;
+		const feedId = ownProps.post && ownProps.post.feed_ID;
+		const siteId = ownProps.post && ( ownProps.post.is_external ? null : ownProps.post.site_ID );
```

Comment:
fun fact… if we provide a default value to `ownProps.props` then we can remove much of this type-checking code

```js
( state, { post: { feed_ID: feedId, is_external, site_ID } = {} } ) => {
	const siteId = is_external ? null : site_ID;

	return Object.assign(
		{ teams: getReaderTeams( state ) },
		feedId > 0 && { feed: getFeed( state, feedId ) },
		siteId > 0 && { site: getSite( state, siteId ) },
	);
}
```

quick note on the return there is that it will produce an object with only those properties which exist whereas when we set them as `undefined` we create an object with three properties whose values may be `undefined`